src/screens/About.js

```diff
@@ -40,6 +41,27 @@ export class About extends React.Component {
     }
   }
 
+  renderBuildVersion() {
+    if (Platform.OS === 'android') {
+      // Android
+      return (<Text style={this.style.text}>{`v${VersionNumber.appVersion} (build ${VersionNumber.buildVersion})`}</Text>);
```

Comment:
Maybe create a method that receive version and build and return this html? It would help us if we decide to change this in the future, so we don't need to change in 3 different places.

Reply:
This is why I created the `renderBuildVersion` method, to isolate this logic, do you think this should be a helper?

Reply:
Not a helper in the project, just the `Text` line could be in a different method, if we want to change the display in the future for any reason. I don't think it's critical